### PR TITLE
[Cloud Security] Add cspm/kspm icons

### PR DESCRIPTION
**Pull request by orouz**
## What does this PR do?

Adds icons for policy templates, so in total we'd have 3 icons including one for the top package. 

## Related issues

- Closes https://github.com/elastic/security-team/issues/5631

## Screenshots
Screenshot 2023-02-07 at 13 06 49
Screenshot 2023-02-07 at 13 06 41

### CODE REVIEW

File: packages/cloud_security_posture/changelog.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # newer versions go on top
-- version: "1.2.10-next"
+- version: "1.2.11"
```
Comment (orouz) [inline packages/cloud_security_posture/changelog.yml:2]: @kfirpeled 

not sure what should this be?

**Comment by orouz** [comment]
CI is failing with this for all 3 icons: 
```
relative path is invalid, target doesn't exist or it exceeds the file size limit
```

- i'm using the latest release and locally `check/lint` both pass, as does `build` 
- those files do exists
- one of them is a bit big, but don't think they pass the limit (not sure what it is)

File: packages/cloud_security_posture/changelog.yml (hunk 1 above)
Comment (kfirpeled) [reply packages/cloud_security_posture/changelog.yml:2]: the `next` is a placeholder for the next release, so we can pack mutilple PRs into a single release

File: packages/cloud_security_posture/manifest.yml [hunk 2]
```diff
@@ -39,8 +39,8 @@ screenshots:
     size: 3160x1708
     type: image/png
 icons:
-  - src: /img/cis-kubernetes-benchmark-logo.svg
-    title: CIS Kubernetes Benchmark logo
+  - src: /img/logo_cloud_security_posture.svg
```
Comment (kfirpeled) [inline packages/cloud_security_posture/manifest.yml:42]: it seems you missed pushing the actual icons

**Review by kfirpeled — APPROVED** [review]

**Pull request merged**